src/coreclr/scripts/jitrollingbuild.py

```diff
@@ -458,6 +466,32 @@ def upload_blob(file, blob_name):
     #        pdb_paths = [os.path.join(pdb_dir, item) for item in os.listdir(pdb_dir) if re.match(r'.*clrjit.*', item)]
     #        files += pdb_paths
 
+    # Figure out which git hash to use for the upload. By default, it is the required coreclr_args.git_hash argument.
+    # However, if "--use_latest_jit_change" is passed, we look backwards in the git log for the nearest git commit
+    # with a JIT change (it could, and often will be, the same as the argument git_hash).
+    jit_git_hash = coreclr_args.git_hash
+
+    if coreclr_args.use_latest_jit_change:
+        # Do all the remaining commands, including a number of 'git' commands including relative paths,
+        # from the root of the runtime repo.
+
+        with ChangeDir(coreclr_args.runtime_repo_location):
+            # Enumerate the last change, starting with the jit_git_hash, that included JIT changes.
+            command = [ "git", "log", "--pretty=format:%H", jit_git_hash, "-1", "--", "src/coreclr/jit/*" ]
+            print("Invoking: {}".format(" ".join(command)))
+            proc = subprocess.Popen(command, stdout=subprocess.PIPE)
```

Comment:
why not use `run_command` from jitutil.py`?

Reply:
I copied/modified this code from elsewhere in jitrollingbuild.py. Looks like jitrollingbuild.py never uses run_command. In fact, it currently doesn't use jitutil at all. That might be a worthwhile (separate) cleanup at some point.